src/main/java/application/rest/v1/CustomResourceWatcher.java

```diff
@@ -137,9 +148,56 @@ private static void setLoggerLevel(JsonObject o) {
                             if (level.equals("all"))
                                 Logger.setLogLevel(Logger.LogLevel.ALL);    
                         }
+                        
+                        //check for ui setting
+                        JsonElement levelUi  = loggingObj.get("ui"); 
+                        if (UI_LOG_LEVEL_API != null && levelUi != null && levelUi.isJsonPrimitive())  {               
+                            String level = levelUi.getAsString();
+                            
+                            if(!CURRENT_UI_LOG_LEVEL.equals(level)) {
```

Comment:
Seems that this wouldn't work with "warning", "none" and "entry" since those values get changed by setUILogLevel().

Reply:
good catch, I'll post a fix